BD-3560: Condense beacon and geofence support info (#8237)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/locations_and_geofences/location_tracking.md

}
```

## Partnership support for beacon and geofence

> Combining existing beacon or geofence support with our targeting and messaging features allows you to learn more about your user's physical actions and message them accordingly. You can leverage location tracking with some of our partners: 

Reviewer: Minor suggested reword for this bit:
```suggestion
Combining existing beacon or geofence support with our targeting and messaging features gives you more information about your users' physical actions so you can message them accordingly. You can leverage location tracking with some of our partners: 
```
